tests/sqllogictests/suites/mode/cluster/distributed_copy_into_table2.test

```diff
@@ -48,4 +48,4 @@ select block_count from fuse_snapshot('default','products');
 2
 
 statement ok
-set enable_distributed_copy_into = 0;
\ No newline at end of file
+set enable_distributed_copy_into = 0;
```

Comment:
can we avoid this modify?

Reply:
It is automatically generated by sqllogictest.